hack/build/build-via-appstudio.sh

```diff
@@ -51,4 +51,5 @@ if [ -z "$COMPONENT" ]; then
 else
   create-component $COMPONENT
 fi
-echo "Run this to show running builds: tkn pr list"
+echo "Run this from physical cluster to show running builds:
+oc get pipelineruns --all-namespaces"
```

Comment:
This might be irrelative for now, since it was made to get pipelineruns easily when kcp is enable. If `tkn pr list` is still a preferred way to list pipelinesruns, I'll revert it.